src/main/java/com/box/sdk/BoxDeveloperEditionAPIConnection.java

```diff
@@ -116,19 +121,21 @@ public static BoxDeveloperEditionConnection getAppAuthConnection(String enterpri
     }
 
     /**
-     * Creates a new Box Developer Edition connection for an App User.
-     * @param userId             the user ID to use for an App User.
-     * @param clientId           the client ID to use when refreshing the access token.
-     * @param clientSecret       the client secret to use when refreshing the access token.
-     * @param privateKey         the private key corresponding to the public key configured with Box Developer Edition.
-     * @param privateKeyPassword the password for the private key.
+     * Creates a new Box Developer Edition connection with App User token.
+     * @param userId                the user ID to use for an App User.
+     * @param clientId              the client ID to use when exchanging the JWT assertion for an access token.
+     * @param clientSecret          the client secret to use when exchanging the JWT assertion for an access token.
+     * @param privateKey            the private key corresponding to public key configured with Box Developer Edition.
+     * @param privateKeyPassword    the password for the private key.
+     * @param  encryptionAlgorithm  specifies the encryption algorithm for generating JWT
```

Comment:
Extra space between `@param` and `encryptionAlgorithm`. Also needs a period at the end of the description.